recipes/area-detector-handlers/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "area-detector-handlers" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "676cec048d2d41136d758fcdfe94a53aca551d2d3c267daf33c4a112a5662f18" %}
```

Comment:
@kbeeperez, while you are here fixing other suggestions, could you please move this hash to the source section below? Thanks.